aiohttp/client.py

```diff
@@ -15,6 +15,7 @@
     Awaitable,
     Callable,
     Coroutine,
+    Final,
```

Comment:
Please import `Final` from `typing_extensions` for backward compatibility with 3.6 and 3.7